apps/studio/src/components/CmsSidebar/CmsSideNav.tsx

```diff
@@ -0,0 +1,192 @@
+import { Suspense } from "react"
+import { useRouter } from "next/router"
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Flex,
+  Skeleton,
+  Text,
+  useDisclosure,
+  VStack,
+} from "@chakra-ui/react"
+import { Menu } from "@opengovsg/design-system-react"
+import { ResourceType } from "@prisma/client"
+import { BiData, BiFile, BiFolder, BiHomeAlt } from "react-icons/bi"
+
+import { trpc } from "~/utils/trpc"
+
+export const CmsSideNav = () => {
+  const { onToggle } = useDisclosure()
+
+  return (
+    <Flex flexDir="column">
+      <VStack
+        align="flex-start"
+        spacing="1.5rem"
+        pt="2rem"
+        px="1.25rem"
+        pb="0.75rem"
+        boxShadow="sm"
+      >
+        <Menu>
+          {({ isOpen }) => (
+            <>
+              <Menu.Button
+                w="full"
+                onClick={onToggle}
+                isOpen={isOpen}
+                variant="outline"
+              >
+                Create new
+              </Menu.Button>
+              <Menu.List>
+                <Menu.Item>Create new page</Menu.Item>
+                <Menu.Item>Create new folder</Menu.Item>
+                <Menu.Item>Create new collection</Menu.Item>
+              </Menu.List>
+            </>
+          )}
+        </Menu>
```

Comment:
I think we do not actually need the internal state of the menu, and may be able to simplify this part:

```suggestion
        <Menu>
          <Menu.Button w="full" variant="outline">
            Create new
          </Menu.Button>
          <Menu.List>
            <Menu.Item>Create new page</Menu.Item>
            <Menu.Item>Create new folder</Menu.Item>
            <Menu.Item>Create new collection</Menu.Item>
          </Menu.List>
        </Menu>
```

Reply:
donezxc